plugins/chat/config/locales/server.en.yml

```diff
@@ -73,6 +73,7 @@ en:
       over_chat_max_direct_message_users:
         one: "You can only create a direct message with yourself."
         other: "You can't create a direct message with more than %{count} other users."
+      root_message_not_found: "The ancestor of the message you are replying cannot be found or has been deleted."
```

Comment:
I would recommend we use `original_message` here, this is what we used after in the code

Reply:
Yep good point will fix